errors/invalid-getstaticprops-value.md

```diff
@@ -1,18 +1,18 @@
 # Invalid getStaticProps Return Value
 
-#### Why This Error Occurred
+## Why This Error Occurred
 
 In one of the page's `getStaticProps` the return value had the incorrect shape.
 
-#### Possible Ways to Fix It
+## Possible Ways to Fix It
```

Comment:
```suggestion
#### Possible Ways to Fix It
```
We use this level of heading in the other error pages so this change isn't needed